sky/cli.py

```diff
@@ -4419,15 +4419,6 @@ def local():
 def local_up():
     """Creates a local cluster."""
     cluster_created = False
-    # Check if ~/.kube/config exists:
-    if os.path.exists(os.path.expanduser('~/.kube/config')):
-        # Check if kubeconfig is valid, `kind delete` leaves an empty kubeconfig
-        valid, reason = kubernetes_utils.check_credentials()
-        if valid or (not valid and 'Invalid configuration' not in reason):
-            # Could be a valid kubeconfig or a non-empty but non-functioning
-            # kubeconfig - check if user wants to overwrite it
-            prompt = 'Cluster config found at ~/.kube/config. Overwrite it?'
-            click.confirm(prompt, default=True, abort=True, show_default=True)
```

Comment:
If a context != `kind-skypilot` is already set, can we print a message saying that we will automatically switch kubectl context?

Reply:
Valid point, I will change the script to prevent switching current context if it's not the SkyPilot context. Wdyt?

Reply:
Hmm, I think it's okay to automatically switch context, just that we should let the user know that we are going to switch contexts.

Reply:
Sounds good, will update 👍

Reply:
Updated in https://github.com/skypilot-org/skypilot/pull/2253/commits/0b10ef96b4d1fdcf68f4099649b40d64ac08ea37